src/init.cpp

```diff
@@ -1632,8 +1632,12 @@ bool AppInitMain(boost::thread_group& threadGroup, CScheduler& scheduler)
     // ********************************************************* Step 11: start node
 
     //// debug print
-    LogPrintf("mapBlockIndex.size() = %u\n",   mapBlockIndex.size());
-    LogPrintf("nBestHeight = %d\n",                   chainActive.Height());
+    {
```

Comment:
I guess it would make sense to also cover `mapBlockIndex` and `chainActive` to make this move-safe.

Reply:
Indeed, ight as well pull chainActive.Height() into the cs_main here.